go/pkg/array/reduce_test.go

```
@@ -0,0 +1,546 @@
+			if val > acc {
+				return val
+			}
+			return acc
+		}, math.Inf(-1))
+
+		require.Equal(t, 4.67, max)
+	})
+
+	t.Run("finds minimum value", func(t *testing.T) {
+		values := []int{5, 3, 8, 1, 9, 2, 7}
+		min := Reduce(values, func(acc, val int) int {
```

golangci-lint (2.2.2)

variable min has same name as predeclared identifier

(predeclared)